client/web/src/hooks/auth.ts

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useEffect, useState } from "react"
+import { apiFetch } from "src/api"
+
+export type AuthResponse = {
+  ok: boolean
+  authUrl?: string
+  error?: string
```

Comment:
remove this also, since it's removed from the go struct?

Reply:
whoops, good catch!